### PR TITLE
Add QNX support to the sdk build scripts

### PR DESCRIPTION
**Pull request by s1341**

### CODE REVIEW

File: Makefile.sdk.mk [hunk 1]
```diff
@@ -191,6 +191,9 @@ ifeq ($(host_platform), linux)
 	v8_host_flags := -f make-linux
 	v8_libs_private := " -lm"
 endif
+ifeq ($(host_platform), qnx)
+	v8_host_flags := -f make-qnx -D snapshot=off
```
Comment (oleavr) [inline Makefile.sdk.mk]: Is `snapshot=off` needed?

**Comment by oleavr** [comment]
Awesome stuff. Thanks!

File: Makefile.sdk.mk (hunk 1 above)
Comment (s1341) [inline Makefile.sdk.mk]: I don't think of it actually does anything. We can probably nuke it. I'll try to check it agian tomorrow.

Comment (oleavr) [inline Makefile.sdk.mk]: That would be great! If it works with it on (the default), it'll improve startup-performance quite a bit.

**Pull request merged**